styles/components/_buttons.scss

```diff
@@ -238,19 +238,28 @@
   &:not(.allowOverflow) {
     overflow: hidden; // clear the floats to avoid layout issues.
   }
-
-  & > .btn {
+  
+  .btn {
```

Comment:
I think there was a situation where we had a nested button in some button strips, and that's why it had the > selector on it. 

But I don't think we do any more. I'm just putting this down as a reminder to myself if we notice a problem later.

Reply:
Oh yeah, sorry... meant to point this one out to you. I didn't want to limit it to immediate children because if you render in a backbone view in there (like I had to), the button will be further nested.

If we do run into the problem you mention, we'll need to come up with some way to manage that case and this case.